stripe-buffer.el

```diff
@@ -136,9 +174,25 @@ Used by `stripe-table-mode' Only the first matching group will be painted."
                                     (goto-char end)
                                     (point))))))
                       ( overlay (apply get-overlay-create-function stripe-region)))
-                 (overlay-put overlay 'face stripe-highlight-face)
-                 (overlay-put overlay 'is-stripe t)
-                 (push overlay sb/overlays)))))
+
+                 (cond
+                  ((unless (require 'dired-subtree nil t)
```

Comment:
If `dired-subtree` is loaded,  `dired-subtree--get-ov` is defined, and `(featurep 'dired-subtree)` returns `t` .
 This branch never uses `dired-subtree--get-ov` as long as load  `dired-subtree`